mne/utils/tests/test_numerics.py

```diff
@@ -465,3 +465,19 @@ def test_julian_conversions():
 
         assert (jd == _dt_to_julian(dd))
         assert (jd == _cal_to_julian(cal[0], cal[1], cal[2]))
+
+
+def test_grand_average_empty_sequence():
+    with pytest.raises(ValueError, match='Please pass a list of Evoked'):
+        grand_average([])
+
+
+def test_grand_average_len_1():
+    base_dir = op.join(op.dirname(__file__), '..', '..', 'io', 'tests',
+                       'data')
+    fname = op.join(base_dir, 'test-ave.fif')
```

Comment:
by convention these go at the top of the file. It helps with consistency and also helps us know what files are required by each test file (to some extent)

Reply:
Incidentally, the name of the file I need to access was already defined at the top level, so will use that variable now